ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/NodeCvesOverviewPage.tsx

```diff
@@ -1,12 +1,13 @@
 import React, { useEffect } from 'react';
```

Comment:
Most of the reordering/sorting in the top-section of this file happened automatically in my IDE. I hope it is okay this way.

Reply:
It is better this way. :)